web/sass/_project.scss

```diff
@@ -272,13 +223,12 @@ label {
         display: block;
     }
     float: right;
-    margin-top: 0.2em;
-    margin-right: 145px;
+    margin-top: 0.4em;
 
     a {
         display: inline-block;
         margin-right: 1em;
-        font-size: 0.8125em;
+        font-size: emCalc(18);
```

Comment:
PR description says 15, and 15 looks centred, 18 does not.

Reply:
Yup, 18 deffo looks wrong. Not sure what happened there. Will fix.